#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("r", double_t, 0, "Radius", 3.0, 0.01, 30.0)
gen.add("min_phi", double_t, 0, "Minimum phi", 0.0, 0.0, pi)
gen.add("max_phi", double_t, 0, "Maximum phi", pi, 0.0, pi)
gen.add("scan_range", double_t, 0, "Range of scan", 2 * pi * 3 / 4, 0.0, 2 * pi)
gen.add("scan_num", int_t, 0, "Sampling number of scans", 1081, 0, 2000)
gen.add("fps", double_t, 0, "fps", 40, 1.0, 100)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "SphericalPointCloudSimulator"))
